teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
```

Comment:
#### The same here :point_down: 

- [ ] Kindly note that as this class enherits from Persone class its constructor should extend all parent's @instance variables comment in the children initialize parameter. :point_down: 

```rb
class Teacher < Person
    def initialize(age, specialization, name = "Unknown", parent_permission = true)
       super(age, name, parent_permission)
       @specialization = specialization
     end
end
```